docs/API.md

```diff
@@ -2762,4 +2762,5 @@ try {
 
 ## 5. Explore Further
 
+- [Build your own Photo API Service - Full Application Example ](https://github.com/minio/minio-java-rest-example)
 - [Complete JavaDoc](http://minio.github.io/minio-java/io/minio/MinioClient.html)
```

Comment:
First of all, this link and the corresponding link in `../docs/zh_CN/API.md` needs to match.
Since this is supposed to point to complete Minio JavaDoc, this link is supposed to be:
`http://minio.github.io/minio-java/`

Reply:
Fixed.